zest/releaser/pypi.py

```diff
@@ -407,6 +408,33 @@ def version_levels(self):
             return default
         return result
 
+    def tag_format(self):
+        """Return the string format for the tag name used in the release.
+
+        Override the default ``%(version)s`` in ~/.pypirc or setup.cfg using
+        a ``tag-format`` option::
+
+            [zest.releaser]
+            tag-format = v%(version)s
+
+        Returns default of ``%(version)s`` when nothing has been configured.
+        """
+        default = '%(version)s'
+        if self.config is None:
+            return default
+        try:
+            result = self.config.get(
+                'zest.releaser', 'tag-format')
+            #test the formater
+            result % ({'version':''})
```

Comment:
This does not actually catch an error. If the format is `foo` the code above won't complain:

```
>>> 'foo' % {'version': ''}
'foo'
```

Trying it out is a good idea, but currently it doesn't help. I don't really know a better way at the moment.